samples/Tests/ColumnsDifferentWidths.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Can you re-run the UWP test app on these and checkin the baseline files? Thanks!